test/release/examples/benchmarks/shootout/meteor-fast.chpl

```diff
@@ -269,13 +269,14 @@ proc searchParallel(in board=0, in pos=0, used=0, placed=0, firstPiece=0) {
       if allMasks[currentMask] {
         const mask = allMasks[currentMask];
 
-        const currentSolution: [piecesDom] int =
-          [firstPiece, mask, 0, 0, 0, 0, 0, 0, 0, 0];
 
-        begin with (in board, in pos)
+        begin with (in board, in pos) {
+          const currentSolution: [piecesDom] int =
+            [firstPiece, mask, 0, 0, 0, 0, 0, 0, 0, 0];
```

Comment:
I ran it twice in each configuration
* 0.03845, 0.04365 without this change
* 0.03531, 0.04106 with this change

That looks "in the noise" to me but might be a slight benefit.